jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -603,6 +603,7 @@ private static void ensureProperties(boolean isInitialization) {
 
 	/*[IF JAVA_SPEC_VERSION >= 18]*/
 	initializedProperties.put("os.version", sysPropOSVersion); //$NON-NLS-1$
+	initializedProperties.put("jdk.reflect.useDirectMethodHandle", "false"); //$NON-NLS-1$
```

Comment:
Is this intended to be temporary? If so, a comment would be welcome.

Reply:
Temporary. A comment has been added.